cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/messaging/aws/sqs/SqsQueueResource.java

```diff
@@ -118,12 +117,8 @@ private void doSendRequest(final SendMessageRequest sendMessageRequest) {
                 return amazonSqsClient.receiveMessage(receiveMessageRequest).getMessages();
             } catch (final AmazonServiceException amazonServiceException) {
                 logger.error("Error receiving SQS messages on queue:[" + queueName + "]", amazonServiceException);
-                // Ignore service errors (5xx) and hope SQS recovers
-                if (amazonServiceException.getErrorType().equals(ErrorType.Service)) {
-                    Thread.sleep(SQS_SERVICE_ERROR_PAUSE_MILLIS);
-                } else {
-                    throw amazonServiceException;
-                }
+                // Ignore all amazon errors and hope SQS recovers
```

Comment:
It is true this exception was thrown but it is actually caught and handled on line 452 : AmazonHttpClient.java. This is then exchanged for a AmazonClientException which is only not being handled in the old code because it is to high up in the inheritance chain. I will commit the new change.